src/containers/gui.jsx

```diff
@@ -85,6 +85,15 @@ class GUI extends React.Component {
                 this.props.onProjectError(error);
             });
     }
+    handleShowExtension(isFromButtonClick = false) {
+        if(isFromButtonClick) {
+            this.props.showExtension();
+        }
+        else if(this.props.projectId === '0' && !this.props.modalChooseExtensionAlreadyBeenOpened) {
```

Comment:
How about using `isExtensionModalOpened` instead of such a big variable name?

Reply:
Hi Biplob, it is good, I would add one word to make more meaningful like `isExtensionModalAlreadyOpened` because I want to check if the modal has been already opened since default project loaded, and not if the modal is open. What do you think?

Reply:
@aquino-luane 
For this case, I will use `isExtensionModalOpened`. If the modal is open, I'll use `isExtensionModalOpen`. However, for clarity, you can use `isExtensionModalAlreadyOpened`, which is the most important.